server/channels/app/user.go

```diff
@@ -843,6 +931,8 @@ func (a *App) UpdateDefaultProfileImage(c request.CTX, user *model.User) *model.
 	return nil
 }
 
+// SetDefaultProfileImage sets the user's profile image to the default image and notifies other users of the change.
```

Comment:
I think this method has better naming, eg. `UpdateDefaultProfileImageAndNotify`.